setup.py

```diff
@@ -75,7 +75,7 @@ def get_test_requires():
       install_requires=[
           'setuptools',
           'lxml >= 2.2',
-          'zope.testbrowser >= 4.0',
+          'zope.testbrowser >= 4.0, < 5.0',
```

Comment:
Wanted to start on a clean slate first. Working on Python 3 on a branch: https://github.com/zopefoundation/z3c.etestbrowser/tree/python3 (Already achieved compatibility with new API of `zope.testbrowser` 5.0.0 😀 )